.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+
+sudo: false  # use the new container-based Travis infrastructure
+os:
+  - osx
+  - linux
+install:
+  - curl -sS http://storage.bos.xamarin.com/bot-provisioning/PortableReferenceAssemblies-2014-04-14.zip > /tmp/pcl-assemblies.zip
+  - unzip /tmp/pcl-assemblies.zip -d /tmp/pcl-assemblies && mv /tmp/pcl-assemblies/PortableReferenceAssemblies-2014-04-14 /tmp/pcl-assemblies/.NETPortable
```

Comment:
But it works on OSX and fails on linux. The same code. Do you know why? 

And when I include PCL it does not fail every time.